src/core/core.scale.js

```diff
@@ -502,6 +502,7 @@ module.exports = function(Chart) {
 			var tickFontFamily = helpers.getValueOrDefault(optionTicks.fontFamily, globalDefaults.defaultFontFamily);
 			var tickLabelFont = helpers.fontString(tickFontSize, tickFontStyle, tickFontFamily);
 			var tl = gridLines.tickMarkLength;
+			var borderDash = gridLines.borderDash;
```

Comment:
for completeness it might be a good idea to also support the `borderDashOffset` which translates to https://developer.mozilla.org/en/docs/Web/API/CanvasRenderingContext2D/lineDashOffset